### PR TITLE
export mapfoldl mapfoldr

### PR DESCRIPTION
**Pull request by rened**

### CODE REVIEW

**Comment by rened** [comment]
Both functions are mentioned in the docs, but they are not exported.

File: base/exports.jl [hunk 1]
```diff
@@ -779,6 +779,8 @@ export
     length,
     map!,
     map,
+	mapfoldl,
```
Comment (nalimilan) [inline base/exports.jl]: Please use spaces instead of tabs.

**Comment by nalimilan** [comment]
Thanks! Could you mention the # of the issue you opened in the commit message? (Also you'd better put your first comment in that message too.)

**Comment by rened** [comment]
Done, thanks for your patience, still learning.

**Comment by nalimilan** [comment]
No problem. Now, please squash the two commits (e.g. using `git rebase -i HEAD~2`).

**Comment by jiahao** [comment]
Build failure seems unrelated.

**Pull request merged**